dbt_project.yml

```diff
@@ -24,6 +26,10 @@ models:
         - level_modeling
         - proj_labels
 
+    enrich:
+      +schema: enrich
```

Comment:
I think enrich is not a straight forward name. We could use common/enrich in staging tables.

 Could we name those derived models' schema like 'erc20/erc721/erc1155' like what Dune did?